packages/auto-approve/src/check-pr.ts

```diff
@@ -163,7 +165,8 @@ function runVersioningValidation(addtlValidationFile?: {
     majorBump = isMajorVersionChanging(versions);
     minorBump = isMinorVersionUpgraded(versions);
     oneDependencyChanged = isOneDependencyChanged(addtlValidationFile.file!);
+    weekdayMerge = mergesOnWeekday();
```

Comment:
nit: I think this would read better if we called this method, `isReleaseDay()`, and `releaseDay` (for the variable respectively). I think this explains what's going on well.

Reply:
I actually prefer the current language, weekday.  If I'm reading the code, "release day" tells me nothing.  "Week day" gives me the right idea, even if it isn't perfectly exact.